lib/JSON/Validator/OpenAPI/Mojolicious.pm

```diff
@@ -19,6 +19,28 @@ has version => 2;
 
 sub E { JSON::Validator::Error->new(@_) }
 
+has generate_definitions_path => sub {
+  my $self = shift;
+  Scalar::Util::weaken($self);
+
+  return sub {
+    my $ref = shift;
+    my @path;
+
+    # Try to determin the path from the fqn
+    if ($ref->fqn =~ m|^(.*)#/(.+)$|) {
```

Comment:
Not a big deal, but can you swap out "|" with "!"?

Reply:
fixed in upcoming push